### ISSUE TITLE
flathub verification?

### ISSUE THREAD

**Issue by qoijjj** [flathub/org.audacityteam.Audacity#147]
Getting verification would allow an increased level of user trust.

https://discourse.flathub.org/t/how-do-you-get-a-flathub-submission-verified/3714
https://flathub.org/apps/collection/verified/1

I believe this would require the blessing of upstream devs.

**Comment by hfiguiere** [comment]
Upstream blessing would probably require to add the telemetry. That's not really user trust.

**Comment by qoijjj** [comment]
> Upstream blessing would probably require to add the telemetry. That's not really user trust.

It does not. Per the link I provided, it just requires upstream to add a single file to their repo that flathub uses to validate that upstream has approved.

**Comment by hfiguiere** [comment]
nothing we can do on this side.

**Comment by cassidyjames** [comment]
I agree there's nothing we can do on this side; I've opened [a discussion on the upstream tracker](https://github.com/audacity/audacity/discussions/5892), but this can probably be closed here.
